src/style.css

```
@@ -1,3 +1,81 @@
 body {
+  --blue-color: #00acdc;
+
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.moyo-header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  justify-content: space-between;
+  text-align: center;
+  padding-inline: 50px;
+  max-width: 100vw;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.moyo-nav {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.box-nav {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  justify-content: end;
+}
+
+.items-nav {
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+  height: 60px;
+  line-height: 60px;
+  display: inline-block;
+  margin-right: 20px;
+  text-align: center;
+}
+
+.box-nav a:hover {
```

try to avoid tag selectors in css
```suggestion
.box-nav .items-nav:hover {
```